components/script/dom/htmlinputelement.rs

```diff
@@ -1110,6 +1122,10 @@ impl VirtualMethods for HTMLInputElement {
         if let Some(ref s) = self.super_type() {
             s.bind_to_tree(tree_in_doc);
         }
+        // TODO: well impletment attr-input-value
+        // https://html.spec.whatwg.org/multipage/#attr-input-value
+        // sanitize when it is first parsed
+        self.sanitize_value();
```

Comment:
This should be fixed.

Reply:
@KiChjang  Do you know where `bind_to_tree` spec is?

Reply:
@tigercosmos Not sure if this is what you want?
https://github.com/servo/servo/blob/0e62a5829b7c29ae2667a21a439aff1e89201bf3/components/script/dom/htmlsourceelement.rs#L47-L54

Reply:
@CYBAI  This is wrong. We have conversation on IRC. You can check it.

Reply:
@CYBAI  The new commit has fixed this part.

Reply:
Got it. Thanks.